### PR TITLE
correct statement on return value of module write hook

### PR DESCRIPTION
**Pull request by boegel**
fix incorrect statement I introduced in #714 🙊

### CODE REVIEW

**Review by akesandgren — APPROVED** [review]
LGTM

File: docs/Hooks.rst [hunk 1]
```diff
@@ -127,7 +127,7 @@ Do take into account the following:
    * the ``EasyBlock`` instance used to perform the installation (usually referred to as ``self``)
    * the filepath of the module that will be written
    * the module text as a string
-  The return value of this hook (if any) will then be appended to the contents of the module file.
+  The return value of this hook (if any) is used as contents for the module file that will be written.
```
Comment (Flamefire) [inline docs/Hooks.rst:130]: TBH: I find "as contents for the file that will be written" a bit to complicated compared to "will be written to the file"
Maybe we could still clarify this a bit, especially with respect to "no return value/no hook":

```suggestion
  The return value of this hook, when set, will replace the original text that is then written to the module file.
```

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

File: docs/Hooks.rst (hunk 1 above)
Comment (boegel) [reply docs/Hooks.rst:130]: That's better indeed, opening another PR :)

Comment (boegel) [reply docs/Hooks.rst:130]: see #716

**Pull request merged**